DEBUG 2021-03-29 10:51:08 __init__.py: 241] CACHEDIR=/home/bao/.cache/matplotlib
DEBUG 2021-03-29 10:51:08 font_manager.py: 1436] Using fontManager instance from /home/bao/.cache/matplotlib/fontlist-v330.json
INFO 2021-03-29 10:51:08 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2021-03-29 10:51:08 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2021-03-29 10:51:08 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:08 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:08 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2021-03-29 10:51:08 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:09 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2021-03-29 10:51:09 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:09 FaceDetModelLoader.py: 36] Successfully loaded the face detection model!
INFO 2021-03-29 10:51:09 FaceDetModelLoader.py: 36] Successfully loaded the face detection model!
INFO 2021-03-29 10:51:09 FaceDetModelLoader.py: 36] Successfully loaded the face detection model!
INFO 2021-03-29 10:51:09 FaceDetModelLoader.py: 36] Successfully loaded the face detection model!
INFO 2021-03-29 10:51:18 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: models, model category: face_alignment，model name: face_alignment_1.0
INFO 2021-03-29 10:51:18 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:18 FaceAlignModelLoader.py: 29] Successfully loaded the face landmark model!
INFO 2021-03-29 10:51:18 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: models, model category: face_alignment，model name: face_alignment_1.0
INFO 2021-03-29 10:51:18 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:18 FaceAlignModelLoader.py: 29] Successfully loaded the face landmark model!
INFO 2021-03-29 10:51:18 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: models, model category: face_alignment，model name: face_alignment_1.0
INFO 2021-03-29 10:51:18 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:19 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: models, model category: face_alignment，model name: face_alignment_1.0
INFO 2021-03-29 10:51:19 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2021-03-29 10:51:19 FaceAlignModelLoader.py: 29] Successfully loaded the face landmark model!
INFO 2021-03-29 10:51:19 FaceAlignModelLoader.py: 29] Successfully loaded the face landmark model!
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IHDR' 16 13
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IDAT' 41 65536
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IHDR' 16 13
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IDAT' 41 65536
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IHDR' 16 13
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IDAT' 41 65536
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IHDR' 16 13
DEBUG 2021-03-29 10:51:26 PngImagePlugin.py: 141] STREAM b'IDAT' 41 65536
DEBUG 2021-03-29 10:53:32 __init__.py: 241] (private) matplotlib data path: /home/bao/.local/lib/python3.8/site-packages/matplotlib/mpl-data
DEBUG 2021-03-29 10:53:32 __init__.py: 241] matplotlib data path: /home/bao/.local/lib/python3.8/site-packages/matplotlib/mpl-data
DEBUG 2021-03-29 10:53:32 __init__.py: 241] CONFIGDIR=/home/bao/.config/matplotlib
DEBUG 2021-03-29 10:53:32 __init__.py: 1480] matplotlib version 3.3.4
DEBUG 2021-03-29 10:53:32 __init__.py: 1481] interactive is False
DEBUG 2021-03-29 10:53:32 __init__.py: 1482] platform is linux
